### PR TITLE
Add displayed log of A8

### PR DESCRIPTION
**Pull request by JianXiongZ**

### CODE REVIEW

File: version.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def version(ip):
                     return None
         try:
             tn.read_until(flag)
-            tn.write('opkg list | grep ^cgminer | cut -c11-\n')
+            tn.write('cat /etc/avalon_version\n')
```
Comment (qinfengling) [inline version.py:56]: @JianXiongZ NOTE: If you upgrade the cgminer package, then the version should not match the /etc/avalon_version.

File: cgminer.php [hunk 2]
```diff
@@ -385,7 +413,32 @@ function second2human($seconds) {
...
+					echo "LED ON</button></td>";
+				else
+					echo "LED OFF</button></td>";
+				echo $td . "<button onClick=\"reboot_mm('" . $ip . "'," . $port . "," . substr($stat['ID'], 3) . "," . $mod . ");\">" . "Reboot</button></td>";
+				echo $td . second2human($matches[3]) . "</td>";
+				echo $td . substr($stat['ID'],0,3) . "-" . substr($stat['ID'], 3) . "-" . $mod . "</td>";
+				echo $td . $matches[1] . "</td>";
+				echo $td . substr($matches[2], 12) . "</td>";
+				echo $td . number_format($matches[4], 0, ".", ",") . "</td>";
+				#echo $td . $matches[6] . "%" . "</td>";
```
Comment (qinfengling) [inline cgminer.php:431]: @JianXiongZ Remove the useless code please.

**Comment by JianXiongZ** [comment]
So, we want to show which information or display all ?
 
 
------------------ Original ------------------

**Comment by qinfengling** [comment]
@JianXiongZ Is this need to merge ?

**Pull request closed without merging**